### PR TITLE
Add karma for JS tests

### PR DESCRIPTION
**Pull request by PVince81**
@DeepDiver1975 can you check travis.yml ? I have the feeling there is some kind of conflict between using env variables in the included matrix vs what is at the top of the file.

### CODE REVIEW

**Comment by PVince81** [comment]
rebased

**Comment by phil-davis** [comment]
@PVince81 there are unresolved conflict lines still in the diff:
```
<<<<<<< HEAD
```

**Comment by PVince81** [comment]
and funnily enough Travis still hasn't finished after all these days...

**Comment by PVince81** [comment]
fixed travis file, let's see...

File: .travis.yml [hunk 1]
```diff
@@ -43,8 +43,20 @@ before_install:
   - ./occ app:list
 
 script:
-  - cd apps/$APP_NAME/tests/acceptance
-  - ./run.sh
+  - cd apps/$APP_NAME
+  - make $TEST
+
+matrix:
```
Comment (phil-davis) [inline .travis.yml]: Now there are 2 matrix include sections.

**Comment by patrickjahns** [comment]
@ownclouders rebase

**Comment by PVince81** [comment]
removed second matrix section

**Comment by PVince81** [comment]
green, can we merge this ?

**Comment by patrickjahns** [comment]
@ownclouders rebase

**Comment by PVince81** [comment]
likely not enabled on this repo

**Comment by PVince81** [comment]
rebased

**Comment by PVince81** [comment]
what ?!

```
Package operations: 118 installs, 0 updates, 0 removals
  - Installing ocramius/package-versions (1.3.0): Downloading (100%)
PHP Parse error:  syntax error, unexpected ':', expecting ';' or '{' in /home/travis/build/owncloud/core/lib/composer/ocramius/package-versions/src/PackageVersions/Installer.php on line 59
Parse error: syntax error, unexpected ':', expecting ';' or '{' in /home/travis/build/owncloud/core/lib/composer/ocramius/package-versions/src/PackageVersions/Installer.php on line 59
make: *** [lib/composer/phpunit] Error 255
```

**Comment by PVince81** [comment]
I'll wait for the Drone CI bootstrap before rebasing this @patrickjahns 

You might want to already include this. If not, I can try and rebase later to include the JS tests.

**Comment by patrickjahns** [comment]
closing in favor of #210

**Pull request closed without merging**